File: python/model/inventario.py
class Inventario:

    # constructor
    def __init__(self, id_item, nombre_item, cantidad):

        # atributos privados
        self.__id_item = id_item
        self.__nombre_item = nombre_item
        self.__cantidad = cantidad

    # getter id
    @property
    def id_item(self):
        return self.__id_item

    # setter id
    @id_item.setter
    def id_item(self, id_item):
        self.__id_item = id_item

    # getter nombre
    @property
    def nombre_item(self):
        return self.__nombre_item

    # setter nombre
    @nombre_item.setter
    def nombre_item(self, nombre_item):

        # validación
        if len(nombre_item) > 0:
            self.__nombre_item = nombre_item

    # getter cantidad
    @property
    def cantidad(self):
        return self.__cantidad

    # setter cantidad
    @cantidad.setter
    def cantidad(self, cantidad):

        # evita negativos
        if cantidad >= 0:
            self.__cantidad = cantidad

File: java/src/model/Inventario.java
package model;

public class Inventario {

    // atributos privados
    private int idItem;
    private String nombreItem;
    private int cantidad;

    // constructor
    public Inventario(int idItem, String nombreItem, int cantidad) {

        this.idItem = idItem;
        this.nombreItem = nombreItem;
        this.cantidad = cantidad;
    }

    // getter id
    public int getIdItem() {
        return idItem;
    }

    // setter id
    public void setIdItem(int idItem) {
        this.idItem = idItem;
    }

    // getter nombre
    public String getNombreItem() {
        return nombreItem;
    }

    // setter nombre
    public void setNombreItem(String nombreItem) {

        // validación
        if(nombreItem.length() > 0){
            this.nombreItem = nombreItem;
        }
    }

    // getter cantidad
    public int getCantidad() {
        return cantidad;
    }

    // setter cantidad
    public void setCantidad(int cantidad) {

        // evita negativos
        if(cantidad >= 0){
            this.cantidad = cantidad;
        }
    }
}